static/src/javascripts/projects/commercial/modules/prebid/utils.js

```diff
@@ -128,6 +128,20 @@ export const shouldIncludePangaea = (): boolean =>
     config.get('switches.ozonePangaea') &&
     config.get('page.section', '').toLowerCase() === 'technology';
 
+export const shouldIncludeXaxis = (): boolean => {
+    const hasFirstLook =
+        config.get('page.isDev') || getRandomIntInclusive(1, 10) === 1;
+    if (config.get('page.edition') === 'UK') {
+        return hasFirstLook;
+    }
+    return false;
+};
+
+export const shouldIncludeImproveDigital = (): boolean => {
+    const edition: ?string = config.get('page.edition');
+    return edition === 'UK' || edition === 'INT';
+};
+
```

Comment:
I think these could be geotargeted, but probably best to do in a separate PR.

Reply:
Agreed! Ideally want to remove all references to config.page.edition from Prebid 😁

Added this as a card here: https://trello.com/c/PQixSSBg